README.md

```diff
@@ -257,6 +257,18 @@ bzk.Command("job", "actions on jobs", func(cmd *cli.Cmd) {
     cmd.Command("log", "show a job log", nil)
 })
 ```
+When you just want to set Action to cmd, you can use ActionCommand function for this
+```go
+app.Command("list", "list all configs", ActionCommand(func() { list() }))
```

Comment:
`ActionCommand` needs to be prefixed with the package name `cli`.

Also, this should also work:

```go
app.Command("list", "list all configs", cli.ActionCommand(list))
```